packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -210,34 +210,42 @@ export function Campaigns() {
         <div className={styles.root}>
             <div className={styles.filters}>
                 <TextInput
-                    className={styles.filterInput}
+                    className={{
```

Comment:
Here we should keep a single string ad a className, and use the following way to style parts of a component:

```css
.custom :global(.part) {
    @apply ...
}
```